paper-radio-group.html

```diff
@@ -45,6 +46,9 @@
   </style>
 
   <template>
+    <iron-a11y-keys keys="up left" on-keys-pressed="_selectPrevious"></iron-a11y-keys>
+    <iron-a11y-keys keys="down right" on-keys-pressed="_selectNext"></iron-a11y-keys>
```

Comment:
This is certainly an acceptable usage of the element. However, as we explored the new world, we generally moved away from using the element form of discrete functionality for Performance Reasons™. It may be better in the long run to use the behavior form, rather than create two additional non-visible elements every time a `<paper-radio-group>` is made. What do you think about this?